src/das-employer-favourites.ApplicationServices/Commands/GetApprenticeshipFavouriteRequestHandler.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using DfE.EmployerFavourites.ApplicationServices.Domain;
+using MediatR;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.EAS.Account.Api.Client;
+
+namespace DfE.EmployerFavourites.ApplicationServices.Commands
+{
+    public class GetApprenticeshipFavouriteRequestHandler : IRequestHandler<GetApprenticeshipFavouritesRequest,ApprenticeshipFavourites>
+    {
+        private readonly ILogger<GetApprenticeshipFavouriteRequestHandler> _logger;
+        private readonly IFavouritesRepository _repository;
+        private readonly IEmployerAccountRepository _employerAccountRepository;
+
+        public GetApprenticeshipFavouriteRequestHandler(
+            ILogger<GetApprenticeshipFavouriteRequestHandler> logger,
+            IFavouritesRepository repository, IEmployerAccountRepository employerAccountRepository)
+        {
+            _logger = logger;
```

Comment:
*_logger* isn't used currently. Should we be logging what accountId has had a request made somewhere?